runtime/polkadot/src/weights/runtime_parachains_disputes.rs

```diff
@@ -16,11 +16,11 @@
 //! Autogenerated weights for `runtime_parachains::disputes`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-02-25, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2022-03-11, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("polkadot-dev"), DB CACHE: 1024
 
 // Executed Command:
-// target/production/polkadot
+// target/release/polkadot
```

Comment:
This does not look good.

See 	[#[pallet::weight(T::SystemWeightInfo::remark(_remark.len() as u32))]](https://github.com/paritytech/polkadot/blob/975e780ae0d988dc033f400ba822d14b326ee5b9/scripts/run_benches_for_runtime.sh#L8)